### ISSUE TITLE
Problems with day/time picker for low vision.

### ISSUE THREAD

**Issue by barbarer** [RunestoneInteractive/rs#821]
From the PI workshop in June 2025.

Low Vision/accessibility issue - I cannot edit the time via the textbox AND as my default text size is large and I zoom in to read the text, when I scroll it exits the date/time picker:

<img width="625" alt="Image" src="https://github.com/user-attachments/assets/82eee85f-6b83-438d-9a05-31fd8423a765" />
